Makefile

```diff
@@ -82,8 +82,11 @@ ifneq ($(HAS_SERVER),)
 		echo "golangci-lint is not installed. Please see https://github.com/golangci/golangci-lint#install-golangci-lint for installation instructions."; \
 		exit 1; \
 	fi; \
+
 	@echo Running golangci-lint
-	cd server && golangci-lint run ./...
+	# cd server && golangci-lint run ./...
```

Comment:
Remove comments.

Reply:
Oops, thanks for catching! I've been having trouble running golangci-lint in this repository and shouldn't have committed this.